src/sentry/integrations/gitlab/client.py

```diff
@@ -348,7 +348,11 @@ def get_file(self, repo: Repository, path: str, ref: str, codeowners: bool = Fal
         )
         contents = self.get(request_path, params={"ref": ref}, raw_response=True, headers=headers)
 
-        result = contents.content if codeowners else b64decode(contents["content"]).decode("utf-8")
+        result = (
+            contents.content.decode()
```

Comment:
Should this be `decode("utf-8")`?

Reply:
I think this works either way, but I can add it in another PR

Reply:
here's the follow up PR: https://github.com/getsentry/sentry/pull/62986